lib/screens/explore_screens/vouch/sponsor_tab/interactor/viewmodels/sponsor_bloc.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:seeds/datasource/remote/model/member_model.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+
+part 'sponsor_event.dart';
+
+part 'sponsor_state.dart';
```

Comment:
Remove line space between state-event imports (just because I like in that way and all rest blocs follow that style).

Reply:
Hi Raul, when I format the page it give that extra space.

Reply:
I guess it is a matter of the Android Studio, leave it like that and I will correct it later haha